### PR TITLE
EWI: Lost To follow up

### PR DESCRIPTION
**Pull request by solemabrothers**
@ssmusoke please find the updated pull request here, sorry for the trouble caused...

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetUpLostToFollowUp.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+import org.openmrs.module.ugandaemrreports.reporting.library.cohort.ARTCohortLibrary;
+import org.openmrs.reporting.data.DatasetDefinition;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Properties;
+@Component
+public class SetUpLostToFollowUp extends UgandaEMRDataExportManager {
```
Comment (ssmusoke) [inline api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetUpLostToFollowUp.java]: @solemabrothers This file should be named EWILostToFollowup

File: omod/src/main/resources/apps/ugandaemr_reports_app.json [hunk 2]
```diff
@@ -315,7 +318,17 @@
         "label": "Family Connect EMTCT Module Data Export",
         "url": "reportingui/runReport.page?reportDefinition=bf79f017-8591-4eaf-88c9-1cde33226517",
         "order": 5
-      },{
+      },
+      {
+        "id": " EWI Lost To Follow Up",
+        "extensionPointId": "org.openmrs.module.ugandaemr.ewi",
+        "type": "link",
+        "label": " EWI Lost To Follow Up",
```
Comment (ssmusoke) [inline omod/src/main/resources/apps/ugandaemr_reports_app.json]: No need for EWI in the label

File: omod/src/main/webapp/pages/reports.gsp [hunk 3]
```diff
@@ -95,7 +97,22 @@
             </div>
         </div>
         <% } %>
-    </div>
+        <% if (ewi) { %>
```
Comment (ssmusoke) [inline omod/src/main/webapp/pages/reports.gsp]: @solemabrothers put EWI below MER indicators as EWI is used less

File: omod/src/main/resources/apps/ugandaemr_reports_app.json (hunk 2 above)
Comment (solemabrothers) [reply omod/src/main/resources/apps/ugandaemr_reports_app.json]: Ressolved

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetUpEWILostToFollowUp.java [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+    public String getExcelDesignUuid() {
+        return "3e7363e9-33d8-4b6c-b290-0314d1910c87";
+    }
+    @Override
+    public String getUuid() {
+        return "e66d26f2-baf9-4b07-bbb8-335e4258ced0";
+    }
+    @Override
+    public String getName() {
+        return " EWI Lost To Follow Up";
```
Comment (ssmusoke) [inline api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetUpEWILostToFollowUp.java]: @solemabrothers Change to "Early Warning Indicators - Lost to Followup"

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetUpEWILostToFollowUp.java [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+    }
+    @Override
+    public String getDescription() {
+        return "EWI Lost To Follow Up";
```
Comment (ssmusoke) [inline api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetUpEWILostToFollowUp.java]: @solemabrothers Change to "Early Warning Indicators - Lost to Followup"

**Pull request merged**